linux/introduction.tw2

```diff
@@ -0,0 +1,14 @@
+::Linux Introduction
+# Installing Exercism on Linux
+
+To install Exorcism on Linux you need to get the `exorcism`
```

Comment:
s/Exorcism/Exercism/ :)

also s/exorcism/exercism/

(twice on the same line, I think you win!)

Reply:
Ooops. :blush: